bot/utils/helpers.py

```diff
@@ -30,3 +31,11 @@ def has_lines(string: str, count: int) -> bool:
 def pad_base64(data: str) -> str:
     """Return base64 `data` with padding characters to ensure its length is a multiple of 4."""
     return data + "=" * (-len(data) % 4)
+
+
+def join_role_stats(role_ids: List[int], name: str, guild: Guild) -> Dict[str, int]:
+    """Return a dict object with the number of `members` of each role given, and the `name` for this joined group."""
+    members = []
+    for role_id in role_ids:
+        members += guild.get_role(role_id).members
```

Comment:
`guild.get_role(role_id)` can return `None`, leading to the following error:
```python
bot_1       | Traceback (most recent call last):
bot_1       |   File "/usr/local/lib/python3.9/site-packages/discord/ext/commands/core.py", line 85, in wrapped
bot_1       |     ret = await coro(*args, **kwargs)
bot_1       |   File "/bot/bot/exts/info/information.py", line 194, in server_info
bot_1       |     member_counts = self.get_member_counts(ctx.guild)
bot_1       |   File "/bot/bot/exts/info/information.py", line 57, in get_member_counts
bot_1       |     **join_role_stats([constants.Roles.project_leads, constants.Roles.domain_leads], "Leads", guild))
bot_1       |   File "/bot/bot/utils/helpers.py", line 40, in join_role_stats
bot_1       |     members += guild.get_role(role_id).members
bot_1       | AttributeError: 'NoneType' object has no attribute 'members'
```
This is also said to the user:
![image](https://user-images.githubusercontent.com/15021300/126910238-ebdaa426-0d32-40be-88cb-55d7362eb865.png)

Reply:
Okay, I pushed some changes and this should not be a problem anymore.